packages/salesforcedx-vscode-lightning/src/telemetry/telemetry.ts

```diff
@@ -8,7 +8,7 @@
 import * as util from 'util';
 import * as vscode from 'vscode';
 import TelemetryReporter from 'vscode-extension-telemetry';
-import { telemetryService } from '.';
+// import { telemetryService } from '.';
```

Comment:
No idea how this happened, I feel like this was a bad merge somewhere along the line - dunno how it was working before